### ISSUE TITLE
Problem retrieving a page file within a zip folder on Windows

### ISSUE THREAD

**Issue by vineetbansal** [Princeton-CDH/ppa-django#271]
I'm using Windows to run the `index` manage command, and ran into an issue at line:

https://github.com/Princeton-CDH/ppa-django/blob/f3d524e21a194120189703ff83119e229d2a3ba6/ppa/archive/models.py#L670

Because of `os.path.join` the zipfile module was looking for a path like (as one example):
```
39015003633594\\00000001.txt
```

Turns out according to the zip file format specification, forward slashes are used as path separators within a zip file, regardless of `os.sep`. So changing the code to the something like the following fixed this issue:
```
pagefilename = self.hathi_content_dir + '/' + page.text_file_location
```

Not sure if this is the cleanest solution though.

**Comment by rlskoeser** [comment]
Thanks for reporting this @vineetbansal - none of us are using Windows so we never tripped over it, but it should be easy to fix now that we know.

**Comment by rlskoeser** [comment]
@vineetbansal I implemented a fix for this that's now in the develop branch, could you check it out and confirm if it works for you now?

**Comment by vineetbansal** [comment]
Hmm..I'm not seeing the fix in the `develop` branch:
https://github.com/Princeton-CDH/ppa-django/blob/develop/ppa/archive/models.py#L645

Also, switching to the `develop` branch just before running `python manage.py index` is causing a
```
ImportError: No module named 'requests_oauthlib'
```
which seems like a new requirement for `ppa/archive/hathi.py` that I had to add in dev-requirements.txt

**Comment by thatbudakguy** [comment]
@vineetbansal re: the `ImportError` I got this too recently; it's actually already in `requirements.txt` - a quick
```sh
pip install -r requirements.txt
```
should take care of it.

**Comment by rlskoeser** [comment]
@vineetbansal wow, you're right - I thought sure I checked that fix into develop, I'm confused about how it got lost. I could only see it on a feature branch that has already been merged, and that commit wasn't part of the PR commit history... 

Here's the fix, should now be showing up in develop: https://github.com/Princeton-CDH/ppa-django/blob/develop/ppa/archive/models.py#L644-L645 

I'm glad to see I did add `requests_oauthlib` to requirements.txt. We haven't come up with a good way to signal when requirements have changed for in-progress development.

Thank you @vineetbansal for taking a look at this! I'm glad you caught that the fix wasn't actually where I thought it was.
